ethers-core/src/types/chain.rs

```diff
@@ -270,9 +272,9 @@ impl Chain {
             FilecoinHyperspaceTestnet | FilecoinMainnet => 30_000,
 
             // Explicitly exhaustive. See NB above.
-            Morden | Ropsten | Rinkeby | Goerli | Kovan | XDai | Chiado | Sepolia | Moonbase |
-            MoonbeamDev | Optimism | OptimismGoerli | OptimismKovan | Poa | Sokol | Rsk |
-            EmeraldTestnet | Boba => return None,
+            Morden | Ropsten | Rinkeby | Goerli | Kovan | XDai | Chiado | Sepolia | Moonbase
+            | MoonbeamDev | Optimism | OptimismGoerli | OptimismKovan | Poa | Sokol | Rsk
+            | EmeraldTestnet | Boba | ZKSync => return None,
```

Comment:
please use the nightly toolchain to format your code
(cargo +nightly fmt)

Reply:
I've run cargo +nigtly fmt in the project